src/index.js

```diff
@@ -21,17 +21,17 @@ import 'babel-polyfill';
 import Mnemonic from 'bitcore-mnemonic';
 import naclFactory from 'js-nacl';
 import crypto from './crypto';
-import dapp from './transactions/dapp';
-import delegate from './transactions/delegate';
-import multisignature from './transactions/multisignature';
-import multisignatureTransaction from './transactions/multisignatureTransaction';
-import signature from './transactions/signature';
-import transaction from './transactions/transaction';
-import inTransfer from './transactions/inTransfer';
-import outTransfer from './transactions/outTransfer';
-import vote from './transactions/vote';
 import api from './api/liskApi';
 import slots from './time/slots';
+import send from './transactions/0_send';
+import registerSecondSignature from './transactions/1_registerSecondSignature';
+import registerDelegate from './transactions/2_registerDelegate';
+import castVotes from './transactions/3_castVotes';
+import createDapp from './transactions/5_createDapp';
+import registerMultisignature from './transactions/4_registerMultisignatureAccount';
+import transferInDapp from './transactions/6_transferInDapp';
+import transferOutDapp from './transactions/7_transferOutDapp';
+import multiSignatureSend from './transactions/multisignatureSend';
```

Comment:
Is there a way to combine `multiSignatureSend` with `send`? If not, can we prefix the file name with `0_` since it's also type 0?

Reply:
Yes, ultimately this should be solved by this https://github.com/LiskHQ/lisk-js/issues/398